src/ProgressOnderwijsUtils/Data/CascadedDelete.cs

```diff
@@ -170,19 +158,38 @@ void DeleteKids(DatabaseDescription.Table table, ParameterizedSql tempTableName,
                     () => {
                         var nrRowsToDelete = SQL($"select count(*) from {tempTableName}").ReadScalar<int>(conn);
                         log($"Delete {nrRowsToDelete} from {table.QualifiedName}...");
+
+                        ParameterizedSql DeletionQuery(ParameterizedSql outputClause)
+                            => SQL($@"
+                                delete pk
+                                {outputClause}
+                                from {table.QualifiedNameSql} pk
+                                join {tempTableName} tt on {ttJoin};
+                            ");
+
+                        DataTable? DeletionExecution()
+                        {
+                            if (outputAllDeletedRows) {
+                                if (table.Triggers.Any()) {
+                                    return SQL($@"
+                                        declare @output_deleted table(
+                                            {table.Columns.Select(col => col.ColumnMetaData.AsStaticRowVersion().ToSqlColumnDefinitionSql()).ConcatenateSql(SQL($", "))}
+                                        );
+                                        {DeletionQuery(SQL($"output {table.Columns.Select(col => SQL($"deleted.{col.SqlColumnName()}")).ConcatenateSql(SQL($", "))} into @output_deleted"))}

```

Comment:
is dit niet 
```suggestion
                                        {DeletionQuery(SQL($"output deleted.* into @output_deleted"))}
```

?

Reply:
ja dat is het wel; zowel `*` als `.Columns` zijn in ColumnId order.

evt. kun je de query dus meer regulier maken door in beide takken waar dit speelt dezelfde specificatie te gebruiken

Reply:
Dan zouden we afhankelijk zijn van de `DatabaseDescription.table.Columns` en dat leek me te link.

Reply:
Ok. hergebruik leek mij hier te klein / onnodig.

Reply:
Het wordt expliciet gesorteerd, en volgens mij is dat ook wel nodig; het zou anders namelijk verwarrend zijn indien kolommen hier en in de systables en in de `select *`s in aparte volgordes zijn.

Maar als je't echt te tricky vindt kun je ook andersom de * vervangen door deze column list.